src/Database/Query.php

```diff
@@ -1393,6 +1394,27 @@ public function into($table)
     }
 
     /**
+     * Creates an expression that refers to an identifier. Identifiers are used to refer to field names and allow
+     * the SQL compiler to apply quotes or escape the identifier.
+     *
+     * The value is used as is, and you might be required to use aliases or include the table reference in
+     * the identifier. Do not use this method to inject SQL methods or logical statements.
+     *
+     * ### Example
+     *
+     * ```
+     * $query->newExp()->lte('count', $query->identifier('total'));
+     * ```
+     *
+     * @param string $identifier The identifier for an expression
+     * @return ExpressionInterface
+     */
+    public function identifier($identifier)
+    {
+        return new IdentifierExpression($identifier);
```

Comment:
@ndm2 no front facing APIs in the core are supposed to explicitly require an internal interface or class. If you find such an example, please let me know and I'll fix it. If you use PhpStorm it will highlight such parts of the code, and I've never had that happen so far.